assets/src/edit-story/output/page.js

```diff
@@ -25,21 +25,16 @@ import PropTypes from 'prop-types';
 import StoryPropTypes from '../types';
 import { PAGE_WIDTH, PAGE_HEIGHT } from '../constants';
 import { generateOverlayStyles, OverlayType } from '../utils/backgroundOverlay';
+import generatePatternStyles from '../utils/generatePatternStyles';
 import OutputElement from './element';
 import getLongestMediaElement from './utils/getLongestMediaElement';
 
 const ASPECT_RATIO = `${PAGE_WIDTH}:${PAGE_HEIGHT}`;
 
 function OutputPage({ page, autoAdvance, defaultPageDuration }) {
-  const { id, elements, backgroundOverlay } = page;
+  const { id, elements, backgroundOverlay, backgroundColor } = page;
   const backgroundStyles = {
-    backgroundColor: 'white',
-    backgroundImage: `linear-gradient(45deg, #999999 25%, transparent 25%),
-      linear-gradient(-45deg, #999999 25%, transparent 25%),
-      linear-gradient(45deg, transparent 75%, #999999 75%),
-      linear-gradient(-45deg, transparent 75%, #999999 75%)`,
-    backgroundSize: '20px 20px',
-    backgroundPosition: '0 0, 0 10px, 10px -10px, -10px 0px',
```

Comment:
What was this doing and is it not needed any more?

Reply:
This was the squared background of a Page. When we have a background for the page directly, then this is not needed. It might actually be a good idea to add the `backgroundColor: 'white'` back just in case as a fallback for any issue.